### ISSUE TITLE
Shooting through the portals work and damages the TARDIS

### ISSUE THREAD

**Issue by parar020100** [MattJeanes/TARDIS#940]
Most likely it is an issue of the doors collision
Shooting out of the interior doors damages the TARDIS

**Comment by parar020100** [comment]
Relates to #479

I'm pretty sure we had the shooting feature working properly before

**Comment by AmyJeanes** [comment]
Yep still seeing this on default interior at least - the tracers and decals look like they're going through but do no damage to NPCs on the other side and it does damage the TARDIS interior.

**Comment by AmyJeanes** [comment]
Fixed in https://github.com/AmyJeanes/world-portals/commit/11fe90f9e977dea280689baec57cae871d54b3c2
